modular_skyrat/modules/borg_buffs/code/robot.dm

```diff
@@ -211,7 +211,7 @@
 
 // Wirebrush for janiborg
 /datum/design/borg_wirebrush
-	name = "Cyborg Upgrade (Wire-brush)"
```

Comment:
You should keep the "Cyborg Upgrade" part only because I could see it being confusing to see like, "Wire-brush Module" and nothing else. Like yeah wire brush module but what the fuck do I put it in.

Reply:
> You should keep the "Cyborg Upgrade" part only because I could see it being confusing to see like, "Wire-brush Module" and nothing else. Like yeah wire brush module but what the fuck do I put it in.

that's the display name inside the borg upgrade section in the mechfab, i'm just unifying the naming so it's not just stupid looking (see below)

Reply:
![image](https://user-images.githubusercontent.com/22140677/212971768-5a896b92-d6e0-481f-b1d6-ff4b099ab6ff.png)
 it already is in the upgrade section for borgs, it doesnt need to say Cyborg Upgrade when no other ones do